core-build/gulp-core-build/src/tasks/jest.schema.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "Jest Task configuration",
+  "description": "Defines Jest task configuration",
+
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "ci": {
+      "description": "Indicate whether the test is running in a CI environment",
+      "type": "boolean"
+    },
+    "configFilePath": {
+      "description": "The path to a jest config file",
+      "type": "string"
+    },
+    "coverage": {
+      "description": "Indicates that test coverage information should be collected and reported in the output",
```

Comment:
>"Indicates that test coverage information should be collected and reported in the output", [](start = 21, length = 90)

Will this break the verify-coverage task?

Reply:
It should not. Jest also uses istanbul and supports istanbul reporters.

---
In reply to: [144159694](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r144159694) [](ancestors = 144159694)